packages/contractkit/src/utils/azure-key-vault-client.ts

```diff
@@ -123,6 +126,14 @@ export class AzureKeyVaultClient {
     return true
   }
 
+  public async getSecret(secretName: string): Promise<string> {
+    const secret = await this.secretClient.getSecret(secretName)
+    if (typeof secret.value === 'undefined') {
```

Comment:
you could check with just `if (secret.value)`
for example and empty string it's a valid one?

Reply:
Technically empty string is valid, but unlikely correct.